### PR TITLE
Transfers integration to the sequencer

### PR DESCRIPTION
**Pull request by MartinOndejka**

### CODE REVIEW

File: src/app/zeko/sequencer/tests/local_network/run.ml [hunk 1]
```diff
@@ -8,13 +8,16 @@ module Graphql_cohttp_async =
...
+  in
   let t =
     Gql.
-      { db =
-          Ledger.Db.create ~directory_name:db_dir
-            ~depth:Gql.constraint_constants.ledger_depth ()
+      { db
       ; slot = Mina_numbers.Global_slot_since_genesis.zero
       ; commands = Hashtbl.create (module String)
+      ; archive = Sequencer_lib.Archive.create ~kvdb:(Ledger.Db.kvdb db)
```
Comment (L-as) [inline src/app/zeko/sequencer/tests/local_network/run.ml:20]: Are you running the archive relay? What is happening concretely?

Comment (MartinOndejka) [reply src/app/zeko/sequencer/tests/local_network/run.ml:20]: No this is the pre-archive relay solution, where the `Archive` stores the events and actions and exposes through the gql.

Comment (MartinOndejka) [reply src/app/zeko/sequencer/tests/local_network/run.ml:20]: I've just added it to the testing local network to be able to test the transfers working.

**Comment by L-as** [comment]
@MartinOndejka

**Comment by MartinOndejka** [comment]
> @MartinOndejka

Will work on this when I finish the DA layer.

**Pull request closed without merging**